BENCHMARKING.md

```diff
@@ -113,22 +116,22 @@ script as well. The behavior of the suite can be modified using environment
 variables. Assuming the `bash` shell is used, these can either be specified via
 the `export` command to persist between multiple runs:
 
-```sh
+```bash
 export VARIABLE="value"
 ...
 make benchmark
 ```
 
-or specified on the fly, on the same line as the `make benchmark` command:
+or specified on the fly, on the same line as the `make benchmark'' command:
```

Comment:
I don't quite understand this change , the others like `bash` and `export` have the same pre- and postfix.

Reply:
Good catch, I actually don't know why this was changed.